spec/controllers/rental_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe RentalController, :type => :controller do
+  describe 'GET index' do
+    it 'has a 200 status code' do
+      get :index
+      expect(response.status).to eq(200)
+      expect(response.data.length).to eq(3)
```

Comment:
If you want to check for `data` in your response, I think the better approach would be to create your test data directly in each test where it's necessary, and then clean your test database in between tests.

In the webapp we use the [database cleaner](https://github.com/DatabaseCleaner/database_cleaner) gem for this -  you can see examples in [this file](https://github.com/simplepractice/simplepractice/blob/staging/spec/spec_helper.rb#L261) of how it's used.